src/lib/messages.js

```diff
@@ -8,26 +8,30 @@ export const introMessage = [
 	"+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~+",
 	"|                                                                             |",
 	"| WebVM is a virtual Linux environment running in the browser via WebAssembly.|",
+	"| WebVM is a virtual Linux environment running in the browser via WebAssembly.|",
```

Comment:
Careful this commit is spurious. All the actual changes are already applied the rest are spurious whitespaces